etc/compile/wait.sh

```diff
@@ -2,8 +2,7 @@
 
 CONTAINER="${1}"
 
-docker wait "$CONTAINER" 2>/dev/null 1>/dev/null
-RET=$?
+RET=`docker wait "$CONTAINER"`
```

Comment:
This was cherry picked from https://github.com/pachyderm/pachyderm/pull/4323 because it was needed to debug issues; see comment here: https://github.com/pachyderm/pachyderm/pull/4323#discussion_r346482117